src/net/sf/memoranda/ui/ExportSticker.java

```diff
@@ -69,12 +69,12 @@ public boolean export(String src){
                         fwrite.write(contents);
                         
                         fwrite.close();
-                        JOptionPane.showMessageDialog(null,Local.getString("Documento creado con exito en su carpeta Memoranda =D"));
+/*TJ*/                        JOptionPane.showMessageDialog(null,Local.getString("Document created with success in its Memoranda folder =D"));

```

Comment:
Code Reviewed by ghgofort (Galen Goforth) using informal code review procedure.  --No defects found in task